sway/tree/container.c

```diff
@@ -781,13 +754,25 @@ void container_damage_whole(struct sway_container *container) {
 	}
 }
 
+/**
+ * Return the output which will be used for scale purposes.
+ * This is the most recently entered output.
+ */
+static struct sway_output *container_get_effective_output(
```

Comment:
Note: in the future we'll want to use the effective output when sending frame events. We don't want to send frame events at 120FPS if a view has entered 2 outputs. That's also how Weston does it.